compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParserErrorHandler.java

```diff
@@ -2297,14 +2309,16 @@ private ParserRuleContext getNextRuleForCloseBrace(int nextLookahead) {
             case FORK_STMT:
                 endContext(); // end fork-statement
                 return ParserRuleContext.STATEMENT;
+            case LIST_CONSTRUCTOR:
+                return ParserRuleContext.CLOSE_BRACKET;
```

Comment:
Can a close-brace be present in a list-constructor context?

Reply:
This is a mistaken addition from a previous commit of mine. I'll fix.